charts/smokey/templates/workflows/smokey-loop.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: argoproj.io/v1alpha1
+kind: CronWorkflow
+metadata:
+  name: smokey-loop
+spec:
+  schedule: "*/3 9-17 * * 1-5"
+  timezone: Europe/London
+  concurrencyPolicy: "Replace"
```

nit: unnecessary quotes
```suggestion
  concurrencyPolicy: Replace
```

---

will do, thanks